.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish packages (stable)
```

Comment:
Any specific reason to have two different files `publish-packages` and `publish-preview` since I see majority of the content is same.